conf/application.conf

```diff
@@ -31,7 +31,6 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuditModule"
```

Comment:
please change uk.gov.hmrc.play.bootstrap.AuditModule to uk.gov.hmrc.play.audit.AuditModule

Reply:
done